pyproject.toml

```diff
@@ -142,4 +200,5 @@ spaces_indent_inline_array = 4
 
 [tool.tomlsort.overrides]
 "project.classifiers".inline_arrays = false
+"tool.pytest.ini_options.filterwarnings".inline_arrays = false
```

Comment:
I'm not familiar with this option. Can you provide some documentation? Also, there are not currently any tests, and pytest is not installed. Is this being used at all?

Reply:
tomlsort sorts toml files. However, we don't want the key `tool.pytest.ini_options.filterwarnings` to be sorted since there, the ordering is important.

Since we don't have any test, I can delete that.